test/conformance/kuttl/exceptions/events-creation/05-assert.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 involvedObject:
-  apiVersion: kyverno.io/v2alpha1
```

Comment:
interesting, did it make a difference ?

Reply:
But it's weird as v2alpha1 is the storage version.

Reply:
Yeah, that's weird 🤔

Reply:
We need to check what is the preferred version, not sure it matches the storage version in this case.

Reply:
Yeah it's the v2beta1.
```
kubectl api-resources | grep policyexceptions
policyexceptions                  polex        kyverno.io/v2beta1                     true         PolicyException
```

Reply:
Hmm, can you check if we can specify the preferred version ourself ?